packages/swirl-components/src/components/swirl-table/swirl-table.tsx

```diff
@@ -228,10 +237,43 @@ export class SwirlTable {
 
       const columnWidth =
         column.width || `${column.getBoundingClientRect().width}px`;
+      const isLastColumnSticky =
+        column.sticky && columns.length === colIndex + 1;
```

Comment:
**note:** The current implementation only allows the last column to be sticky to the right end to the table. All other columns are sticky to the left.